### ISSUE TITLE
App and Thermostat devices improvement

### ISSUE THREAD

**Issue by dnhaas** [athombv/homey#1234]
**What is your Homey version (Settings → System)?**

1.1.2

**What did you try to achieve?**

Scroll through the devices on the app

**What was the result?**

Thermostat changed temperature to 29 degrees

**What did you expect as result?**

Scroll to the bottom of the devices

**Any other remarks**

Its too easy to accidently change the setpoint of a thermostat i.e. the Nest.

Would be better to first push the device to activate it, and than you can swipe to change the setpoint

**Comment by Dijker** [comment]
Exact same on my Thermosmart! Accidently set it to 28 Gr. C and was not happy when I came home..... 
Maybe some kind of lock or first open the thermostat to change it.

**Comment by daneedk** [comment]
This is also very annoying with other cards, for example the Thinking Cleaner or a lamp that can change color. The only way to scroll up and down without accidentally activating or changing something if you only use the left side of the screen (roughly only the part that shows icons) for scrolling. Not a good UX in my opinion.

**Comment by bvdbos** [comment]
I don't agree... Having to first open a device by clicking it and only then be able to set it, is bad UX/UI imho...

**Comment by dnhaas** [comment]
just as the devices that are "greyed out" in this screenshot, you first have to push it once to activate the functions of that device

![greyed out](https://cloud.githubusercontent.com/assets/22715833/21660130/6aca9fe2-d2ce-11e6-9108-848bf95393d5.png)
